clang-tidy issue since #1599 merge

Hi, 
clang-tidy raise some error since the merge of #1599

In a Ubuntu 20 docker image (even though the actual version is irrelevant i think to this case)
```bash
docker run --rm -it --entrypoint bash ghcr.io/shamrock-code/ci-ubuntu20:main
```

clang tidy on a cpp file having just a single include to sycl/sycl.hpp create the following error : 
```
root@885939f463c4:/home/docker# clang-tidy-17 main.cpp  -- -Iacpp/include/AdaptiveCpp
1 warning and 1 error generated.
Error while processing /home/docker/main.cpp.
acpp/include/AdaptiveCpp/hipSYCL/algorithms/util/memory_streaming.hpp:64:27: error: expected namespace name [clang-diagnostic-error]
   64 |     namespace jit = sycl::AdaptiveCpp_jit;
      |                     ~~~~~~^
Suppressed 1 warnings (1 in non-user code).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
Found compiler error(s).
```

The following commands create a clang-tidy error :
```bash
wget https://apt.llvm.org/llvm.sh
chmod +x llvm.sh
sudo ./llvm.sh 17
sudo apt install -y libclang-17-dev clang-tools-17 libomp-17-dev clang-tidy-17

git clone https://github.com/AdaptiveCpp/AdaptiveCpp.git
cmake -G Ninja -S AdaptiveCpp -B AdaptiveCpp/build -DCMAKE_INSTALL_PREFIX=acpp
cd AdaptiveCpp/build
ninja && ninja install
cd ../..

printf "#include<sycl/sycl.hpp>\nint main(){}" > main.cpp
clang-tidy-17 main.cpp  -- -Iacpp/include/AdaptiveCpp
```

Additionally the error can be traced to that specific commit : 
`c07b5023c8c9b7937a29b3b2b2ac754297ed4f18`

--- a/include/hipSYCL/algorithms/util/memory_streaming.hpp
+++ b/include/hipSYCL/algorithms/util/memory_streaming.hpp
@@ -15,6 +15,7 @@
 #include "hipSYCL/sycl/device.hpp"
 #include "hipSYCL/sycl/libkernel/nd_item.hpp"
 #include "hipSYCL/sycl/info/device.hpp"
+#include "hipSYCL/sycl/jit.hpp"
 #include <cstddef>
 
 

